src/derivation.js

```diff
@@ -161,12 +235,22 @@ export const getDerivationModesForCurrency = (
       all.push("segwit_unsplit");
     }
   }
+  if (currency.supportsSegwit) {
+    all.push("segwit_on_legacy");
+    all.push("legacy_on_segwit");
+  }
   all.push("");
   if (currency.supportsSegwit) {
```

Comment:
same confition as above, is this on purpose ?

Reply:
yes because there is a `push("")` between the 2.

the order makes that we scan invalid paths first, then the "normal" `44'` accounts, then the legacy `49'` accounts

Reply:
👍